index.js

```diff
@@ -1,17 +1,70 @@
 class SortedList {
-  constructor() {}
+    constructor() {
+        this.items = [];
+        this.length = this.items.length;
+    }
 
-  add(item) {}
+    add(item) {
+        this.items.push(item);
+        this.items = this.items.sort(function(a, b) { return a - b });
+        this.length = this.items.length;
+        return this.items;
+    }
 
-  get(pos) {}
+    get(pos) {
+        if (this.items[pos]) {
+            return this.items[pos];
 
-  max() {}
+        } else {
+            throw new Error('OutOfBounds');
+        }
 
-  min() {}
+    }
```

Comment:
```
 max() {
    // if the array is empty (!this.length)
    if (this.items.length === 0) {
      throw new Error('EmptySortedList');
    }

    // since we work with sorted array, we can simply point the last element of the array as
    // the highest value of the array
    return this.items[this.length - 1];

    // you can use some more fancy ways as well
    // return Math.max(...this.items);
  }

  min() {
    // if the array is empty (!this.length)
    if (this.items.length === 0) {
      throw new Error('EmptySortedList');
    }

    return Math.min(...this.items);

    // or, since we work with sorted array, we can simply point the first element of
    // the array as the lowest value of the array
    // return this.items[0];
  }
```